### ISSUE TITLE
Unwanted popup scrollbars in Firefox

### ISSUE THREAD

**Issue by ghostwords** [EFForg/privacybadger#1610]
![screenshot from 2017-08-22 17 03 07](https://user-images.githubusercontent.com/794578/29587509-360eca56-875c-11e7-95ae-1d55df325ede.png)

From https://github.com/EFForg/privacybadger/pull/1516#issuecomment-324151917.

**Comment by lemnis** [comment]
I'm unable to reproduce this issue, could you give me some extra info (which site, used OS, etc.) ?

**Comment by ghostwords** [comment]
Firefox 55.0.2 on Linux. Can reproduce by opening Firefox with a fresh profile, installing Privacy Badger from master, visiting `cnn.com` (or any site with enough third-party domains) and opening the popup.

**Comment by ghostwords** [comment]
I can't reproduce on long-enough domain lists anymore, but I can reproduce in the popup on whitelisted sites (screenshot below) and on special browser pages (in the #1737 branch).

![screenshot from 2017-10-19 14 52 48](https://user-images.githubusercontent.com/794578/31788679-72e9d296-b4dd-11e7-8145-dc41d5b5eddc.png)

**Comment by lemnis** [comment]
I tested before in macOS and in elementary os (fork of ubuntu) and couldn't reproduce it. 

It can occur when firefox finished its height calculation before it stop rendering. I only saw this occur when trying the fork in #1445 with flexbox, but I believe it can also occur with `calc()` and 
`display: grid;`.  
Maybe I should create a bug report at bugzilla for above issue, but I didn't bother till now. It's a pretty rare case.

A possible solution could be found while cleaning up popup.css, every time I look at I see css that could be coded neater.
